datarobot_batch_scoring/utils.py

```diff
@@ -54,11 +54,11 @@ def verify_objectid(value):
     OptKey('api_token'): t.String,
     OptKey('create_api_token'): t.String,
     OptKey('pred_name'): t.String,
-    OptKey('skip_row_id'): t.Bool,
+    OptKey('skip_row_id'): t.StrBool,
```

Comment:
oh, cool! I didn't trafaret has `StrBool`

Reply:
I didn't know either :) That was my first time experience with trafaret. Though initially I thought I'd need to either implement my own trafaret or perform type conversion earlier using `ConfigParser.getboolean()` method.